spec/models/ability_spec.rb

```diff
@@ -454,19 +454,11 @@
     let(:user)              { super_admin.user }
     let(:other_super_admin) { create(:super_admin) }
     let(:provider)          { create(:provider) }
-    let(:other_provider)    { create(:provider) }
     let(:external_user)          { create(:external_user, provider: provider)}
-    let(:other_external_user)    { create(:external_user, provider: other_provider)}
 
     it { should be_able_to(:update_settings, user) }
     it { should_not be_able_to(:update_settings, another_user) }
 
-    context 'can manage any provider' do
```

Comment:
probably worth having a negative test to replace this.

```
 context 'cannot manage any provider' do
      [:show, :index, :new, :create, :edit, :update].each do |action|	
        it { should_not be_able_to(action, provider) }	
      end	
    end
```

Reply:
👍